Database/Corrections/sodQuestFixes.lua

```diff
@@ -296,6 +296,10 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.preQuestSingle] = {78506},
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [78561] = {
+            [questKeys.preQuestSingle] = {78537},
+            [questKeys.zoneOrSort] = sortKeys.SHAMAN,
+        },
         [78575] = {
             [questKeys.objectives] = {{{212694}}},
             [questKeys.preQuestSingle] = {78537},
```

Comment:
Part 4 needs the new Part 3 as pre quest:

```suggestion
            [questKeys.preQuestSingle] = {78561},
```

Reply:
Slipped my mind! 

Fixed in: https://github.com/Questie/Questie/pull/5372/commits/50aa1f1e4414da0832b6e2061f3726f0374ac68a